.github/workflows/test.yml

```diff
@@ -78,13 +86,13 @@ jobs:
       run: >-
         python3 -m pip freeze --all
 
-    - name: >-
-        Initialize tox envs
-      run: >-
-        python3 -m tox --notest --skip-missing-interpreters false -vv
-      env: ${{ matrix.env }}
+    - name: Initialize tox envs
+      run: |
+        set -ex
```

Comment:
This is unnecessary because
1) it's just one command that we're running below
2) it is already visible in the log, no need to print it out twice